src/AnsiUtils.ps1

```diff
@@ -35,12 +35,14 @@ function EscapeAnsiString([string]$AnsiString) {
     $res
 }
 
-function Test-VirtualTerminalSequece([psobject]$Object) {
-    if ($global:GitPromptSettings.AnsiConsole -and ($Object -is [string])) {
-        return $Object.Contains($AnsiEscape)
-    }
-    else {
-        return $false
+function Test-VirtualTerminalSequece([psobject[]]$Object, [switch]$Force) {
+    foreach ($obj in $Object) {
```

Comment:
A little confused by this. Are you meaning to `return` based on the first object received?

Reply:
I thought I'd changed that. No, those should simply output a Boolean value.  I'll fix that.